### DISCUSSION TITLE
WebUI Dimmer Slider not updated on command

### DISCUSSION THREAD

**Discussion by Heizelmann** [arendst/Tasmota#17897] [Q&A] [answered]
When I update a dimmer value by command from console or by http request the slider on an open WebUI does not move to the new position, only the on/off state is updated whenn necessary.

**Comment by barbudor** [comment, marked as answer]
Correct
If yu refresh the page or go to another page and come back, the dimmer should update to the correct value

There is no plan to change this behavior.
Web UI is mostly considered as a debug tool more than an everyday operating UI.
